### PR TITLE
json mod entries

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/json-module/terms/json-load/json-load.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+print(type(bar))
+
+eggs = json.loads(bar)
+print(type(eggs))
+# Output: <class 'str'> <class 'dict'>
+```
+
+## Codebyte Example
+
+The `json.load` function can be used to convert a json file to a Python object:
```
Comment (KTom101) [inline content/python/concepts/json-module/terms/json-load/json-load.md:45]:
```suggestion
The `json.load()` function can be used to convert a json file to a Python object:
```

File: content/python/concepts/json-module/terms/json-load/json-load.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+Title: 'json.load()'
```
Comment (KTom101) [inline content/python/concepts/json-module/terms/json-load/json-load.md:2]:
```suggestion
Title: '.load()'
```

File: content/python/concepts/json-module/terms/json-dump/json-dump.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+Title: 'json.dump()'
```
Comment (KTom101) [inline content/python/concepts/json-module/terms/json-dump/json-dump.md:2]:
```suggestion
Title: '.dump()'
```

File: content/python/concepts/json-module/terms/json-load/json-load.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
```
Comment (KTom101) [inline content/python/concepts/json-module/terms/json-load/json-load.md:1]: Please change the folder name from `json-load`--> `load` and file name from `json-load.md` --> `load.md`

File: content/python/concepts/json-module/terms/json-dump/json-dump.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
```
Comment (KTom101) [inline content/python/concepts/json-module/terms/json-dump/json-dump.md:1]: Please change folder name from `json-dump` --> `dump` and file name from `json-dump.md` to `dump.md`

**Review by KTom101 — CHANGES_REQUESTED** [review]
@caupolicandiaz Here's my review of this entry, thanks!

File: content/python/concepts/json-module/terms/json-dump/json-dump.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+```py
+import json
+foo = {'first':'hello', 'second':'world'}
+
+bar = json.dumps(foo)
+print(type(bar))
+
+eggs = json.dumps(bar)
+print(type(eggs))
+# Output: <class 'str'> <class 'dict'>
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/python/concepts/json-module/terms/json-dump/json-dump.md:42]: I double checked this, and I still get the appropriate results. Not sure why you wouldn't

[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/python/concepts/json-module/terms/json-dump/json-dump.md:42]: the second one should've been loads not dumps

**Comment by caupolicandiaz** [comment]
@KTom101 I've updated the files with your comments, should be all set. Thanks

**Review by KTom101 — APPROVED** [review]
@caupolicandiaz Thanks! This looks ready for the next review.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@caupolicandiaz Finished with my review and left some comments/suggestions for you 😄

File: content/python/concepts/json-module/json-module.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `json` module provides tools for working with the json format. It can be imported with the following line:
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/python/concepts/json-module/json-module.md:17]: I believe JSON is correct

**Comment by caupolicandiaz** [comment]
@Dusch4593 all set implementing your comments. Thanks

**Review by Dusch4593 — APPROVED** [review]
@caupolicandiaz LGTM! 👍🏻

**Comment by Dusch4593** [comment]
Given that there are multiple files in this PR, let's give it one more review. @SSwiniarski or @yangc95?

File: content/python/concepts/json-module/json-module.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+---
+Title: 'JSON Module'
+Description: 'The `json` module is used for encoding and decoding objects to and from the JSON format.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
```
Comment (SSwiniarski) [inline content/python/concepts/json-module/json-module.md:10]:
```suggestion
  - 'Strings'
  - 'JSON'
```

File: content/python/concepts/json-module/terms/dump/dump.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
+---
+Title: '.dump()'
+Description: 'Encodes a Python object as a JSON file.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
```
Comment (SSwiniarski) [inline content/python/concepts/json-module/terms/dump/dump.md:10]:
```suggestion
  - 'Strings'
  - `JSON`
```

File: content/python/concepts/json-module/terms/dump/dump.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `.dump()` function encodes a Python object as a JSON file. The encoding conversion is based on the following [table](https://docs.python.org/3/library/json.html#json-to-py-table). The `.dumps()` function, alternatively, takes a Python object and returns a JSON string.
+
+## Syntax
+
+```pseudo
+json.dump(py_obj)
```
Comment (SSwiniarski) [inline content/python/concepts/json-module/terms/dump/dump.md:22]: Are we missing an output file here?

File: content/python/concepts/json-module/terms/load/load.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
+---
+Title: '.load()'
+Description: 'Decodes a JSON file and returns a Python object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
```
Comment (SSwiniarski) [inline content/python/concepts/json-module/terms/load/load.md:10]:
```suggestion
  - 'Strings'
  - 'JSON'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@caupolicandiaz, I reviewed and had a couple of comments.

File: content/python/concepts/json-module/json-module.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `json` module provides tools for working with the Javascript Object Notation (JSON) format. It can be imported with the following line:
```
Comment (yangc95) [inline content/python/concepts/json-module/json-module.md:17]: Javascript -> JavaScript

**Comment by caupolicandiaz** [comment]
@SSwiniarski I've incorporated your comments, should be all set now. Thanks

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
LGTM! 👍

**Comment by Dusch4593** [comment]
> LGTM! 👍

@SSwiniarski did you mean to approve?

**Review by SSwiniarski — APPROVED** [review]
I meant to approve this. Looks good to merge.

**Comment by Dusch4593** [comment]
Yup we are ready to merge! 👍🏻 Made some light edits to `CatalogContent` in 7798534cdf623655b5fed2fa93dde3dcc337f6b3, 322b9c1260f77a91c6afe3b8dff6d859a6d9cce9, and c857fd16193bb1b0bc8f669915fcc445a7519e20.

**Pull request merged**